modules/base/rendering/renderablemodel.cpp

```diff
@@ -390,9 +463,22 @@ RenderableModel::RenderableModel(const ghoul::Dictionary& dictionary)
         _modelTransform = glm::mat4_cast(glm::quat(glm::radians(_rotationVec.value())));
     });
 
+    _enableAnimation.onChange([this]() {
+        if (!_geometry->hasAnimation()) {
+            LWARNING("Attempting to enable animation for a model that does not have any");
+        }
+        else if (_enableAnimation.value() && _animationStart == "") {
```

Comment:
See above re std::string::empty

Reply:
Also the `.value()` call shouldn't be necessary as the BoolProperty has an `operator bool()` function overload